Kernel/Config/Files/XML/Framework.xml

```diff
@@ -9206,8 +9206,8 @@ via the Preferences button after logging in.
             </Array>
         </Value>
     </Setting>
-    <Setting Name="Elasticsearch::ArticleIndexCreationSettings###all" Required="0" Valid="1">
-        <Description Translatable="1">Number of shards (NS) and replicas (NR) for the index.</Description>
+    <Setting Name="Elasticsearch::ArticleIndexCreationSettings" Required="0" Valid="1">
```

Comment:
Maybe I overlooked this somewhere, but could you please look up, what the FieldsLimit actually does, and whether we really want to add it, please? (This is an addition of Tomohisa and would be new with this patch, too.)

Reply:
Personally I don't think we need FieldsLimit by default. This defines the maximum number of fields in an index. The default setting is 1000. It is meant to prevent very large mappings and therefore poorly performing searches.
I did understand Tomohisas introduction here as an example for what is possible.
Maybe we should remove it from our configuration and instead deliver an explained example for a more extended configuration.

Reply:
Forget my previous comment. I decided to make usage of the FieldsLimit setting in the installer. I think, Tomohisa intended do this but did not finished it for whatever reason.

Reply:
Forget my previous comment. I decided to make usage of the FieldsLimit setting in the installer. I think, Tomohisa intended do this but did not finished it for whatever reason.

Reply:
Is there anything speaking against setting it to 1000, if it's the default? Probably it doesn't matter much, but it's just keeping everything exactly the same for people who are currently using it.